openshift/pipelines/Jenkinsfile.cicd

```diff
@@ -36,12 +34,18 @@ pipeline {
     // The name of the project namespace(s).
     NAME_SPACE = "jcxjin"
 
-    // Deployment configurations used within this pipeline
-    DEV_ENV = "dev"
+    // This should be provided by the OpenShift Pipeline Env variables
```

Comment:
Moved these to OpenShift env variables to allow for the creation of multiple pipelines based on configuration.  Otherwise this would cause multiple builds for multiple instances.

Reply:
If this has been tested I'd rather we remove the commented out code - this is tracked in git if we ever want this back.